pkg/spec/createconfig.go

```diff
@@ -310,6 +310,8 @@ func (c *CreateConfig) GetContainerCreateOptions(runtime *libpod.Runtime) ([]lib
 	var options []libpod.CtrCreateOption
 	var portBindings []ocicni.PortMapping
 	var err error
+	var podInfraContainer *libpod.Container
```

Comment:
We should handle this inside `libpod` - add a function to the pod to get the pause container's ID, and add a set of bools inside the container config ("UsePodIPCNs", "UsePodNetNs", etc). This will require some changes to container.Dependencies() to include the pod pause container's ID there if any of these bools are true. Then, when we generate the final spec in generateSpec(), we look up the pause container then and add its namespaces there.